app/controllers/issues_controller.rb

```diff
@@ -129,6 +137,16 @@ def set_affected_nodes
                           .sort_by { |node, _| node.label }
   end
 
+  def set_form_cancel_path
```

Comment:
I was today-years-old when I learned about `link_to "foo", :back 

> Using a :back Symbol instead of an options hash will generate a link to the referrer (a JavaScript back link will be used in place of a referrer if none exists). If nil is passed as the name the value of the link itself will become the name.

https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to

Reply:
I used this previously but I don't like how it only works if you navigated to a page with clicks. For example, if we use :back in the cancel link on #edit, and a user is linked directly to #edit without navigating to it from another view or if they refresh the page while on it, the #edit view cancel link will just link to #edit and the user can't actually "cancel".